packages/circuits/scripts/build.mjs

```diff
@@ -0,0 +1,5 @@
+// check if circom is in the path
+
+// ask the user for config values
+
+// build the circuits
```

Comment:
not finish yet?
Or it is a TODO in another PR?

Reply:
Ah it was just an idea. Like to have an interactive CLI for building circuits and setting config.